src/v2/Components/Artwork/HoverDetails.tsx

```diff
@@ -0,0 +1,82 @@
+import { Box, Pill } from "@artsy/palette"
+import { themeGet } from "@styled-system/theme-get"
+import { compact } from "lodash"
+import { FC } from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import styled from "styled-components"
+import { HoverDetails_artwork } from "v2/__generated__/HoverDetails_artwork.graphql"
+
+interface HoverDetailsProps {
+  artwork: HoverDetails_artwork
+}
+
+const HoverDetails: FC<HoverDetailsProps> = ({ artwork }) => {
+  const labels = compact([
+    artwork.attributionClass?.name,
+    artwork.mediumType?.name,
+  ])
+  const pills = labels.map((label, index) => ({
+    id: `${artwork.internalID}-pill-${index}`,
+    label,
+  }))
+
+  if (pills.length === 0) {
+    return null
+  }
+
+  return (
+    <HoverContainer>
+      {pills.map((pill, index) => {
+        const isLastPill = pills.length - 1 === index
```

Comment:
**question / non-blocking**: Might not be an appropriate use case for this, but whenever I see logic for determining if an item is last to adjust styling/spacing I think of our [Join component](https://palette.artsy.net/elements/join/) in Palette. Could that and a spacer component with right margin be useful here?

Reply:
Good catch! I once again forgot about the existence of this component